pkg/wwan/mm-init.sh

```diff
@@ -34,12 +34,12 @@ mkdir -p /var/run/dbus
 dbus-daemon --system
 echo "D-Bus daemon started"
 
-echo "Starting Udev daemon"
-udevd --debug --daemon 2>/dev/null
 # Apply installed ModemManager udev rules.
+echo "Setting up udev rules"
+mkdir -p /run/udev/rules.d/
+cp /lib/udev/rules.d/* /run/udev/rules.d/
 udevadm control --reload
 udevadm trigger
```

Comment:
What if udev container is slower and udev daemon is not yet running?
Would this fail and should we then repeat couple times?

Reply:
no, if udev is not up yet, there is no problem because once it gets up, the rules will be already there and it will be taken into account... we run `udevadm trigger` after initialize udev as well...

Reply:
OK